pupa/scrape/base.py

```diff
@@ -5,9 +5,10 @@
 from collections import defaultdict, OrderedDict
 
 import scrapelib
+from django.conf import settings
```

Comment:
gotcha, thanks Josh- 

it looks like we just get the logging config from django.conf settings from what I'm seeing, I don't think there's a good reason for it